codechecker_common/cmd_config.py

```diff
@@ -0,0 +1,53 @@
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+import os
+
+from codechecker_common.util import load_json_or_empty
+from codechecker_common import logger
+
+LOG = logger.get_logger('system')
+
+
+def process_config_file(args, subcommand_name):
+    """
+    Handler to get config file options.
+    """
+    if 'config_file' not in args:
+        return {}
+    if args.config_file and os.path.exists(args.config_file):
+        cfg = load_json_or_empty(args.config_file, default={})
+        return cfg.get(subcommand_name, [])
+
+
+def check_config_file(args):
+    """
+    LOG and check about the config file usage.
+
+    If a config file is set but does not exist the program will
+    exit.
+    LOG is not initialized in the process_config_file function yet
+    so we can not log the usage there. Using print will
+    always print out the config file data which can mess up the
+    tests depending on the output.
+    """
+    if 'config_file' not in args:
+        return
+
+    if 'config_file' in args and args.config_file \
+            and not os.path.exists(args.config_file):
+        raise FileNotFoundError(
+            f"Configuration file '{args.config_file}' does not exist.")
+    elif not args.config_file:
+        return
+
+    cfg = load_json_or_empty(args.config_file, default={})
+    if cfg.get("enabled"):
```

Comment:
As far as I remember we removed the `enabled` key from this configuration file, so this file will be always enabled.